source/extensions/filters/network/ext_authz/ext_authz.h

```diff
@@ -27,7 +27,8 @@ namespace ExtAuthz {
 #define ALL_TCP_EXT_AUTHZ_STATS(COUNTER, GAUGE)         \
   COUNTER(total)                                        \
   COUNTER(error)                                        \
-  COUNTER(denied)                                      \
+  COUNTER(denied)                                       \
+  COUNTER(error_allowed)                                \
```

Comment:
error_allowed is a bit misleading because in the proto it [says not denied](https://github.com/envoyproxy/data-plane-api/blob/cb26bddf359e65f9cf3266258092ca51140d3ab7/envoy/config/filter/network/ext_authz/v2/ext_authz.proto#L25) for when the boolean of failure mode allow should be applied.  So when error_allowed is seen, it leads me to believe that it should only be incremented when a status of Error is encountered.

Reply:
i see what you mean. I will rename the counter to `failure_mode_allowed` that way it is clear why the request was allowed to move on.